projects/ngrx.io/content/guide/effects/install.md

```diff
@@ -21,7 +21,7 @@ yarn add @ngrx/effects
 If your project is using the Angular CLI 6+ then you can install the Effects to your project with the following `ng add` command <a href="https://angular.io/cli/add" target="_blank">(details here)</a>:
 
 ```sh
-ng add @ngrx/effects@latest
+ng add @ngrx/effects@latest --minimal false
```

Comment:
Hi, thanks for creating this PR!

I think it would be better to respect the default options while using the command in the docs.
So I would propose to revert this change and to remove step 3 and 4 in the section below.

Reply:
@timdeschryver Hi! Actually, I think, it's not expected behavior. How will the user understand, that he needs to create these files? 🤣 
Now a user has an idea of what he needs to do.

Reply:
@htnpro it's the intended behavior (#2251)

Reply:
@timdeschryver Okey, understood you. Am I closing PR?

Reply:
If you want, you can update the docs and remove points 3 and 4.

![image](https://user-images.githubusercontent.com/28659384/94398207-6f908900-0165-11eb-9d27-dfd029e8255a.png)

Reply:
@timdeschryver sure, I will.